src/main/ui/app/component/AddOnSearch.jsx

```diff
@@ -56,32 +56,45 @@ export default class AddOnSearch extends Component {
     }
 
     render() {
-        return (
-                <div>
-                    Type:
-                    <select onChange={event => this.setType(event.target.value)}>
-                        <option value="">All Types</option>
-                        <option value="OMOD">Module (OMOD)</option>
-                        <option value="OWA">Open Web App (OWA)</option>
-                    </select>
 
-                    <br/>
+const title = (
+<span>Type: {this.state.type}</span>
+);
+        return (
+                <div className="row col-md-12 col-sm-12 col-xs-12 pushdown">
+                    <div>
+    <div className="input-group">
```

Comment:
Indentation throughout here is distracting.
Also, having inconsistent use of blank lines is distracting, e.g. the div around DropdownButton has a blank line at the start but not at the end.